aws-opentelemetry-distro/src/amazon/opentelemetry/distro/aws_opentelemetry_configurator.py

```diff
@@ -302,6 +301,13 @@ def create_exporter(self):
                 endpoint=application_signals_endpoint, preferred_temporality=temporality_dict
             )
         if protocol == "grpc":
+            # pylint: disable=import-outside-toplevel
+            # Delay import to only occur if gRPC specifically requested. Vended Docker image will not have gRPC bundled,
+            # so importing it at the class level can cause runtime failures.
+            from opentelemetry.exporter.otlp.proto.grpc.metric_exporter import (
+                OTLPMetricExporter as OTLPGrpcOTLPMetricExporter,
```

Comment:
Does this mean we can still support grpc protocol if a user specifies it?

Reply:
Yes, assuming `opentelemetry-exporter-otlp-proto-grpc` is installed.

Reply:
Do we plan to call this out to users in our public docs or somewhere similar?